### PR TITLE
Fix the i18n for multi organization application

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/multi_organization.rst.tmpl [hunk 1]
```diff
@@ -95,6 +102,43 @@ The following roles should be created and completed as needed
...
+
+Modify the `Dockerfile` like that:
+
+.. code::
+
+   - RUN build-l10n "<package>"
+   + RUN build-l10n --suffix=-org1 --suffix=-org2 "<package>"
+
+
+This will use the ``lang_urls_suffix`` defined above.
```
Comment (arnaud-morvan) [inline doc/integrator/multi_organization.rst.tmpl:140]: At runtime, I do not really understand what this sentence make here.

File: doc/integrator/multi_organization.rst.tmpl [hunk 2]
```diff
@@ -95,6 +102,43 @@ The following roles should be created and completed as needed
...
+
```
Comment (arnaud-morvan) [inline doc/integrator/multi_organization.rst.tmpl:141]: I understand that we need to run `make update-client-po` in each instance after build with corresponding env file, but it is not clear.

**Review by arnaud-morvan — COMMENTED** [review]
Seems not completely clear for me.

File: doc/integrator/multi_organization.rst.tmpl [hunk 3]
```diff
@@ -95,6 +102,43 @@ The following roles should be created and completed as needed
...
+   .PHONY: update-client-po
+   update-client-po:
+       cd .. && SUFFIX=-org1 INTERFACES=desktop-org1,mobile-org1 \
+           update-po $(USER_ID) $(GROUP_ID) $(LANGUAGES)
+       cd .. && SUFFIX=-org2 INTERFACES=desktop-org2,mobile-org2 \
+           update-po $(USER_ID) $(GROUP_ID) $(LANGUAGES)
+
+This will correctly filter the themes and the layers but not the layer groups.
+
+All available environment variable:
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply doc/integrator/multi_organization.rst.tmpl:123]: no =>
All available environment variable that can be added in the `geoportal/Makefile` file to run the
`update-po` command:

File: doc/integrator/multi_organization.rst.tmpl (hunk 2 above)
Comment (sbrunner) [reply doc/integrator/multi_organization.rst.tmpl:141]: No, the `make update-client-po` will update all the l10n files...

It's in the rule that we call the `update-po` for every instance.

File: doc/integrator/multi_organization.rst.tmpl (hunk 1 above)
Comment (sbrunner) [reply doc/integrator/multi_organization.rst.tmpl:140]: :-/
=>
For the runtime in the ``interfaces_config`` we will use the ``lang_urls_suffix`` to define witch interface
use witch localization file, see how it's defined above.

**Comment by sbrunner** [comment]
@jwkaltz can you have a look on the documentation?

File: doc/integrator/multi_organization.rst.tmpl [hunk 4]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
 - org2-registered
 - org2-intranet
 
+Internationalization
+--------------------
+
+We should have one localization file for each organization.
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:108]: For each organization, a localization file should be created.

File: doc/integrator/multi_organization.rst.tmpl [hunk 5]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
...
+
+The general workflow is:
+- The integrator will run `make update-client-po`.
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:111]: will run -> performs

File: doc/integrator/multi_organization.rst.tmpl [hunk 6]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
...
+- This will run one `update-po` script for each organization with different environment variables.
+- We will get one po file set for each organization with the defined suffix.
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:113]: We will get -> The result is

File: doc/integrator/multi_organization.rst.tmpl [hunk 7]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
...
+- The integrator needs to complete the po files with translations.
+- When we build the config Docker image it will convert all the po file in the corresponding JSON files
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:115]: When the config Docker image is built, all po files are automatically converted to JSON files

File: doc/integrator/multi_organization.rst.tmpl [hunk 8]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
...
+  with the same suffix.
+- At runtime the `dynamic` view will return the appropriated URLs for the translations.
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:117]: At runtime, the `dynamic` view will return the appropriate URLs for the translations.

File: doc/integrator/multi_organization.rst.tmpl [hunk 9]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
...
+       cd .. && \
+           SUFFIX=-test \
+           INTERFACES=desktop-test,mobile-test \
+           THEME_REGEX=.*-test \
+           GROUP_REGEX=.*-test \
+           WMSLAYER_REGEX=.*-test \
+           WMTSLAYER_REGEX=.*-test \
+           update-po $(USER_ID) $(GROUP_ID) $(LANGUAGES)
+
+All available environment variable that can be added in the `geoportal/Makefile` file to run the
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:145]: The environment variables that are used by the update-po command are the following:

File: doc/integrator/multi_organization.rst.tmpl [hunk 10]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
...
+ - ``THEME_REGEX``: Regular expression used to filter the themes.
+ - ``GROUP_REGEX``: Regular expression used to filter the layer groups.
+ - ``WMSLAYER_REGEX``: Regular expression used to filter the WMS layers.
+ - ``WMTSLAYER_REGEX``: Regular expression used to filter the WMTS layers.
+
+Note that the extractor is able to filter themes and layers based on the given interfaces but there is no
+link between layer groups and interfaces.
+
+
+Modify the `Dockerfile` to create all the needed localizations JSON files:
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:158]: localizations -> localization

File: doc/integrator/multi_organization.rst.tmpl [hunk 11]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
...
+
+.. code::
+
+   - RUN build-l10n "<package>"
+   + RUN build-l10n --suffix=-org1 --suffix=-org2 "<package>"
+
+
+When loafing the frontend the `dynamic` view will return the appropriate localizations URLs based on the
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:166]: When loading the frontend, the `dynamic` view will return the appropriate localization URLs ...

File: doc/integrator/multi_organization.rst.tmpl [hunk 12]
```diff
@@ -95,6 +102,72 @@ The following roles should be created and completed as needed
...
+``lang_urls_suffix`` defined in the corresponding interface (in ``interfaces_config``).
+See how it's defined above.
```
Comment (jwkaltz) [inline doc/integrator/multi_organization.rst.tmpl:168]: See the definition above.

**Review by jwkaltz — APPROVED** [review]
See doc suggestions inline

**Pull request merged**